Rest.Models/Product.cs

```diff
@@ -0,0 +1,6 @@
+﻿public class Product

```

Comment:
Same here. Is a class instead of a record.

Reply:
Same here. My bad. I already pushed the changes.